### PR TITLE
Very short introductions: QC

### PR DESCRIPTION
**Pull request by nekrut**
Starting a series of very short tutorials on fundamental NGS concepts

### CODE REVIEW

**Comment by nekrut** [comment]
Not sure whether it fails because of youtube embed, which I really want to keep as is

File: topics/introduction/tutorials/vsi_qc/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+```
+@M02286:19:000000000-AA549:1:1101:12677:1273 2:N:0:23
+CACTACCCGTGTATCTAATCCTGTTTGATACCCGCACCTTCGAGCTTA
++
+--8A,CCE+,,;,<CC,,<CE@,CFD,,C,CFF+@+@CCEF,,,B+C,
+@M02286:19:000000000-AA549:1:1101:15048:1299 2:N:0:23
+CACTACCGGGGTATCTAATCCTGTTCGCTCCCCACGCTTTCGTCCATC
++
+-6AC,EE@::CF7CFF<<FFGGDFFF,@FGGGG?F7FEGGGDEFF>FF
+```
```
Comment (hexylena) [inline topics/introduction/tutorials/vsi_qc/tutorial.md:101]: if you want a side-by-side rendering, [consider using the `code-in` and `code-out` boxes. ](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#code-box)

File: topics/introduction/tutorials/vsi_qc/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+{: .comment}
+
+## What are base qualities?
+
+As we've seen above, FASTQ datasets contain two types of information:
+
+- *sequence of the read*
+- *base qualities* for each nucleotide in the read.
+
+The base qualities allow us to judge how trustworthy each base in a sequencing read is. The following excerpt from an excellent [tutorial](http://chagall.med.cornell.edu/RNASEQcourse/Intro2RNAseq.pdf) by Friederike D&uuml;ndar, Luce Skrabanek, Paul Zumbo explains what base qualities are:
```
Comment (hexylena) [inline topics/introduction/tutorials/vsi_qc/tutorial.md:141]:
```suggestion
The base qualities allow us to judge how trustworthy each base in a sequencing read is. The following excerpt from an excellent [tutorial](http://chagall.med.cornell.edu/RNASEQcourse/Intro2RNAseq.pdf) by Friederike Dündar, Luce Skrabanek, Paul Zumbo explains what base qualities are:
```

File: topics/introduction/tutorials/vsi_qc/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+> 3. Check that the datatype is set to `fastqsanger.gz` (Galaxy usually automatically assigns the right datatype, but check anyway).
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="datatypes" %}
+>
+> 4. Create a paired collection from your data (relevant part beging at 0:57 and and the case of this tutorial we have 4 datasets): 
+>
+>    <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/uZUt9XIHUQo?si=F0qUj76L_lrM7R2j&amp;start=57" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```
Comment (hexylena) [inline topics/introduction/tutorials/vsi_qc/tutorial.md:202]:
```suggestion
>    {% snippet faqs/galaxy/collections_build_list_paired.md %}
```

perhaps you'd like to add this video (which is captioned, good!) to that snippet?

**Review by hexylena — COMMENTED** [review]
> Not sure whether it fails because of youtube embed, which I really want to keep as is

yes i think the video is fine in this case (a small non-required part of the tutorial, as an additional aid for those who need the visuals..)

**Review by bebatut — COMMENTED** [review]
Thanks @nekrut 

What about having this tutorial in the `sequence-analysis` topic together with the other QC tutorial we already have?

There are also probably bits (e.g. FASTQ format explanation) we put as snippets that could reused in both tutorials
What do you think?

File: topics/introduction/tutorials/vsi_qc/tutorial.md (hunk 3 above)
Comment (nekrut) [reply topics/introduction/tutorials/vsi_qc/tutorial.md:202]: I really want to keep the video as is at the moment. I will make a new one later. This is a draft anyway

**Comment by nekrut** [comment]
@bebatut -> I think we need to start a well coordinated effort on updating existing tutorials and purging old content. Realistically, GCC would be a good time point to do it. But we need:

 - go through every topic
 - identify what we keep and what do hide
 - contact authors and request updates (and perhaps update many ourselves)

**Comment by nekrut** [comment]
> > Not sure whether it fails because of youtube embed, which I really want to keep as is
> 
> yes i think the video is fine in this case (a small non-required part of the tutorial, as an additional aid for those who need the visuals..)

So, what causes this linting issue?

**Comment by bebatut** [comment]
> @bebatut -> I think we need to start a well coordinated effort on updating existing tutorials and purging old content. Realistically, GCC would be a good time point to do it. But we need:
> 
> * go through every topic
> * identify what we keep and what do hide
> * contact authors and request updates (and perhaps update many ourselves)

Yes that would be good. We probably also need people dedicated to coordinate the efforts, support authors in writing and updating, etc. We probably also need to provide incentives for contributors and to avoid requesting too much from them.

**Comment by hexylena** [comment]
> I think we need to start a well coordinated effort on updating existing tutorials and purging old content. Realistically, GCC would be a good time point to do it. But we need:

yes, that's a great gcc project. But for now would you consider moving this to sequence-analysis?

> So, what causes this linting issue?

no this is the linting issue. But I'm saying it's acceptable so whomever merges is aware it's OK to ignore the failure here.

**Review by natefoo — APPROVED** [review]
Apologies for this, Anton and I will follow up on the move and any other issues as requested. He's teaching a class with this today and wanted to have this available as a draft.

**Pull request merged**